content/products/index/index.json

```
@@ -1,6 +1,6 @@
 {
   "seo": {
-    "title": "SSW Consulting Services | Over 25 years of experience developing awesome solutions on top of Angular, React, Blazor, .NET, Azure, GitHub, DevOps (was VSTS/TFS), Power Apps, Power Automate, Power BI, SharePoint, Dynamics and SQL Server",
+    "title": " Explore SSW's products built with the latest tech stack.",
```

```suggestion
    "title": "Explore SSW's products built with the latest tech stack.",
```
Cleaned up the space here to fix possible linting issues